src/app/components/header/header.component.css

```diff
@@ -1,5 +1,6 @@
 .header {
     position: fixed;
+    top: 0;
```

Comment:
put the header to be fix on top and have to use top property to keep it on top=0